packages/clay-card/fixtures/user/CardUser.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import ClayCard from '../../src/index';
+
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClaySticker from '@clayui/sticker';
+
+import React from 'react';
+
+const CardUser = (props: any) => (
+	<ClayCard displayType="user">
+		<ClayCard.Header>
+			<div className="aspect-ratio-item aspect-ratio-item-center-middle aspect-ratio-item-fluid card-type-asset-icon">
```

Comment:
I would expect to be using the `AspectRatio` component here.

Reply:
Fixed